src/test/java/io/cdap/plugin/salesforce/etl/SalesforceBatchMultiSourceETLTest.java

```diff
@@ -163,4 +167,60 @@ public void testBlackList() throws Exception {
     Assert.assertEquals(expectedSObjects, actualSObjects);
   }
 
+  @Test
+  public void testValidateSObjects() throws Exception {
```

Comment:
nit: rename the method as `testValidSObjects()` to keep in sync with the other method testInvalidSObjects()